sources/Context.cpp

```diff
@@ -320,6 +328,31 @@ void Context::execute() {
       throw;
     }
     simu->terminate();
+
+    std::ostringstream timelineStream;
+    simu->printTimeline(timelineStream);
+    const boost::filesystem::path timelineFilePath = simu->getTimelineOutputFile();
```

Comment:
No need to add new API in the core as we have the jobs somewhere

Reply:
I'm not sure it's possible to get the constraints, lost equipments and finalStatte output file names from the jobEntry_ attribute